### ISSUE TITLE
"No Block Targeted" causes mod to stop working

### ISSUE THREAD

**Issue by Spuunj** [ChiselsAndBits/Chisels-and-Bits#932]
When working on a build, I'll be holding some bits of a block in my hand, it can be any block, the result is the same, and if I'm accidentally stood too far away a popup will state "No block targeted" or something similar. The issue comes when I approach the block closer and get into a range where I would normally be able to place the bits on the block, the same popup continues to appear without the mod working until I log out and re load the world

For any bug reports please provide the following details as well,
* MC Version: 1.18.2
* C&B Version: 1.2.90
* Do You have Optifine: Yes

**Comment by Serifina** [comment]
One of my players has also reported this bug.

Pack: The Phoenix's Magics and Goodies
MC Version: 1.18.2
C&B Version: 1.2.101

> The whatever bug with Chisels and Bits arose again. It'd work, and then if I ever run out of bits entirely, even if I get more later, the game refuses to let me place any more bits, giving me a "No block is being targeted" error.

(Direct quote from my player)

**Comment by EntityinArray** [comment]
Happened for me too

- Pack: Direwolf20 1.18
- MC Version: 1.18.2
- C&B Version: 1.2.90
- Do You have Optifine: No

**Comment by MattB70** [comment]
I found that I can temporarily fix the issue without a relog by breaking a bit and placing it back, but this is in creative mode.

**Comment by osirisgothra** [comment]
yep this happens with me too, 1.18.2 stoneblock 3 1.2.115 which is what sb3 is currently shipped with Optifine=No Rubidium/Oculus=No CustomMods=No

breaking a bit (even in survival mode) will temporarily fix the problem, but it happens if you use it near stuff like delightful grass (mod grinding utils) or inverted illuminant blocks (simply light) blocks that have special states or do special things
